packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx

```diff
@@ -168,3 +208,82 @@ export function editPatientVitals(
     },
   });
 }
+
+export function exists(...args: any[]): boolean {
```

Comment:
Should this function be called `allExist`?  The difference of 'some' exist vs 'every' exist.

Reply:
Turns out this function is not strictly necessary.